### PR TITLE
add kanban component

### LINKED ISSUE #96: Kanban component for tasks
Tasks Section

This component should be an aggregate of many other components like `status pills`, `single task tile`, `kanban header` and also the `kanban columns`. It should be possible to pass an array of tasks to the component.

### PR DESCRIPTION
**Pull request by DasProffi**
closes #96
Capture

### CODE REVIEW

File: tasks/components/KanbanHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  sortingStatus?: string,
+  sortingLabel?: string,
+  searchValue: string,
+  onSearchChange: (search: string) => void
+}
+
+export const KanbanHeader = ({
+  language,
+  sortingStatus,
+  sortingLabel,
```
Comment (MaxSchaefer) [inline tasks/components/KanbanHeader.tsx:37]: Both props are not in use

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Works great! ✌️
We can improve the UX in the future by replacing some colors and animations.

File: tasks/components/KanbanHeader.tsx (hunk 1 above)
Comment (DasProffi) [reply tasks/components/KanbanHeader.tsx:37]: because we haven't establish a sorting for the status or label? But the ui indicated the possibility

Comment (DasProffi) [reply tasks/components/KanbanHeader.tsx:37]: I will just omit them for now in the function props

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**